templates/advantage/table/_free-trial.html

```diff
@@ -0,0 +1,41 @@
+<div class="col-start-large-2 col-10">
+  {% if mobile %}
+    <hr class="p-separator" />
+  {% endif %}
+  <div class="row">
+    {% if contract['contractInfo']['status'] == "expired" %}
+
+      <div class="p-notification--caution">
+        <p class="p-notification__response">
+         <span class="p-notification__status">Your free trial expired.</span>You will lose access to this Ubuntu Advantage subscription on {{ contract['contractInfo']['grace_period_end'] }}.
```

Comment:
Can't seem to find this in the copy doc. Could we add it there to make changes easier in the future?

Reply:
I don't think any of the "update your subscription" copy made it into the copy doc.
Good call though! We should add them